plugins/nvm/nvm.plugin.zsh

```diff
@@ -1,3 +1,4 @@
 # The addition 'nvm install' attempts in ~/.profile
 
 [[ -s ~/.nvm/nvm.sh ]] && . ~/.nvm/nvm.sh
+[[ -r $NVM_DIR/bash_completion ]] && . $NVM_DIR/bash_completion
```

Comment:
I use only `git`, `nvm` and `ssh-agent` plugins (and don’t use [`compleat`](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/compleat/compleat.plugin.zsh) plugin). And nvm autocomplete works for me.

zsh 5.0.2 (x86_64-apple-darwin13.0), latest oh-my-zsh

But I’m not sure about no-apple zsh. How to check the plugin?